wrappers/flutter/android/src/main/kotlin/com/fitatu/barcodescanner/fitatu_barcode_scanner/FitatuBarcodeScanner.kt

```diff
@@ -47,21 +37,17 @@ class FitatuBarcodeScanner(
      * This is not guarantee that camera will use this resolution. If camera doesn't support this
      * resolution then the closest one will be picked
      */
-    private val targetResolution by lazy { Size(1280, 720) }
-
     private var camera: Camera? = null
     private var cameraProvider: ProcessCameraProvider? = null
     private var options: ScannerOptions? = null
 
     private lateinit var surfaceTexture: SurfaceTexture
-    private var scanResultJob: Job? = null
 
     override fun init(options: ScannerOptions) {
         this.options = options
         barcodeReader.options = BarcodeReader.Options(
             formats = if (options.qrCode) setOf(
-                BarcodeReader.Format.QR_CODE,
-                BarcodeReader.Format.MICRO_QR_CODE
+                BarcodeReader.Format.QR_CODE, BarcodeReader.Format.MICRO_QR_CODE
```

Comment:
a co z innymi formatami? nie musimy ich podawać?

Reply:
To wziąłem z example xzinga. Z tego co rozumiem, to jak nie podamy nic, to obsługuje wszystko